src/NHSD.BuyingCatalogue.Ordering.Domain/Order.cs

```diff
@@ -42,7 +42,7 @@ private Order(OrderDescription orderDescription, Guid organisationId) : this()
 
         public DateTime Created { get; set; }
 
-        public DateTime? Completed { get; private set; }
+        public DateTime? Completed { get; set; }
```

Comment:
As discussed on the phone, this unfortunately is undoing the encapsulation of the Completed field. Really we only want to set this if the order has been marked as completed in the `Complete` method. This hopefully makes it easier to follow as there is only one place this value can be set. So need to use one of the methods e.g. Reflection to set this as part of a test. We already have an example of this in the `OrderItem` class for the `Created` property.